### ISSUE TITLE
How to compile tic80.js and tic80.wasm from source

### ISSUE THREAD

**Issue by jrkirby** [nesbox/TIC-80#1297]
It's clearly possible to compile to a web version of tic80 - wasm compiled with emscripten - one already exists. Yet I can't update my web version from source, because I can't find any targets in the makefile to build these.

Am I missing something?

Are these compilation targets missing for a reason?

**Comment by nesbox** [comment]
Please look at our build scripts https://github.com/nesbox/TIC-80/blob/master/.github/workflows/build.yml#L306
Thank you.

**Comment by jrkirby** [comment]
Is the build.yml supposed to be executed by some other tool?

I tried running 
```
cd build
emcmake cmake -DCMAKE_BUILD_TYPE=$BUILD_TYPE ..
cmake --build . --parallel --config $BUILD_TYPE
cp html/export.html bin/index.html
```
After fixing emscripten installation issues, it seems to execute with no errors. But I still can't find the tic80.js tic80.wasm in the /build/bin or anywhere else.

Should I be setting $BUILD_TYPE to something? I tried MinSizeRel because it's in that build.yml, but that didn't help.

**Comment by RobLoach** [comment]
> Is the build.yml supposed to be executed by some other tool?

GitHub Actions. You can see the build log over at https://github.com/nesbox/TIC-80/runs/1642840031

**Comment by jrkirby** [comment]
Thanks for your help guys. I've managed to build it using github actions. I'd still like to build the wasm version locally though, which I've haven't managed to succeed despite many attempts.

I'm able to build for linux locally with no issues. But when I try to run the commands above, it just builds the linux again, not the emscripten build.

I checked against the github actions environment variables with printenv and set everything that looks relevant to a corresponding thing on my local machine. It still is trying to compile the linux executable.

Someone told me emcmake is supposed to alter cmake so that it sets up a build for emscripten instead of a native build. But this doesn't seem to be happening in my case. Any suggestions to help me compile this locally with emscripten?
